sycl/unittests/Extensions/CommandGraph/Update.cpp

```diff
@@ -26,20 +26,30 @@ TEST_F(CommandGraphTest, UpdatableException) {
   EXPECT_ANY_THROW(ExecGraphNoUpdatable.update(Node));
 }
 
-TEST_F(CommandGraphTest, DynamicParamRegister) {
-  // Check that registering a dynamic param with a node from a graph that was
-  // not passed to its constructor does not throw.
+TEST_F(CommandGraphTest, DynamicObjRegister) {
+  // Check that registering a dynamic object with a node from a graph that was
+  // not passed to its constructor throws.
```

Comment:
This comment doesn't make sense now since it is checking that it doesn't throw. The error for this was removed in #18199 .

Reply:
I think I messed up the rebase. The code on origin still has the comment but slightly changed. I will update to match what is in there.